addOns/custompayloads/src/main/resources/org/zaproxy/zap/extension/custompayloads/resources/Messages.properties

```diff
@@ -1,3 +1,7 @@
+custompayloads.api.view.viewCustomPayloadsCategoryList = Display all Custom Payloads categories.
+custompayloads.api.view.viewCustomPayloadsList = Display all Custom Payloads (category, payload, and enabled state).
+custompayloads.api.view.viewCustomPayloadsList.param.category = The Custom Payloads category for which the payloads should be displayed.
```

Comment:
I'd reduce the "Custom Payloads" references (just to not be repetitive, these will be under the custom payloads already).